frontend/src/entry.client.tsx

```diff
@@ -52,8 +52,8 @@ async function prepareApp() {
 
 const queryClient = new QueryClient({
   queryCache: new QueryCache({
-    onError: (error) => {
-      toast.error(error.message);
+    onError: (error, query) => {
+      if (!query.queryKey.includes("authenticated")) toast.error(error.message);
```

Comment:
this seems like a bit of a hack--is there a better way?

Reply:
Previously we could handle them in the individual queries but the latest versions of TanStack Query remove these callbacks from `useQuery`, so AFAIK there is no straight forward way.

The alternative would be rendering the error toasts on failures manually (but does repeat itself).

@tofarr Do you know of any alternatives?